test/credentials-cli-command.test.ts

```
@@ -0,0 +1,223 @@
+    runOpenshell: (args: string[]) => {
+      calls.push(args);
+      return { status: 0, stdout: "", stderr: "" };
+    },
+    ...bridge,
+  };
+  require.cache[RUNTIME_PATH] = {
+    id: RUNTIME_PATH,
+    filename: RUNTIME_PATH,
+    loaded: true,
+    exports: runtime,
+  } as NodeJS.Module;
```

ESLint

[error] 'NodeJS' is not defined.

(no-undef)